src/sagemaker/fw_utils.py

```diff
@@ -395,8 +397,10 @@ def model_code_key_prefix(code_location_key_prefix, model_name, image):
     Returns:
         str: the key prefix to be used in uploading code
     """
-    training_job_name = sagemaker.utils.name_from_image(image)
-    return "/".join(filter(None, [code_location_key_prefix, model_name or training_job_name]))
+    name_from_image = f"/model_code/{int(time.time())}"
```

Comment:
Can we make it a None here?
The line 403 does allow it to be  None. And it's counter intuitive as the default path is not actually a image related name

Reply:
not sure if I follow, `sagemaker.utils.name_from_image(image)` will always return a value, but `code_location_key_prefix` can be None.

Reply:
as long as its not a pipeline variable, the value will always be overrode by the output from `sagemaker.utils.name_from_image(image)`

Reply:
I see, makes sense. Can we use the existing helper function `def name_from_base(base, max_length=63, short=False)` here 
like `name_from_base("model_code")` to make the naming aligned with other modules' convention?
The sagemaker.utils.name_from_image() is calling the `name_from_base` as well

Reply:
we want the s3 prefix, not `model_code-timestamp` here tho.